api/base/exceptions.py

```diff
@@ -1,5 +1,35 @@
-from rest_framework.exceptions import APIException
+
 from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+def json_api_exception_handler(exc, context):
+    """ Custom exception handler that returns errors object as an array """
+
+    from rest_framework.views import exception_handler
+    response = exception_handler(exc, context)
+
+    # Title removed to avoid clash with node "title" errors
+    acceptable_members = ['id', 'links', 'status', 'code', 'detail', 'source', 'meta']
+    errors = []
+
+    if response is not None:
```

Comment:
Why not `if response:`? Is there a case other than `None` where `response` could be falsy?